### PR TITLE
Add project ideas for glue

### PR DESCRIPTION
**Pull request by astrofrog**
Just a couple of ideas for now, more to follow potentially

### CODE REVIEW

**Comment by dpshelio** [comment]
Feel free to merge it whenever you are done with it :)

**Comment by astrofrog** [comment]
Done for now, I may make some small edits tomorrow but I'd like to see it rendered first

File: _projects/2017/glue/notebook.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: Glue notebook viewers
+desc: Develop Jupyter notebook viewers for glue
```
Comment (Cadair) [inline _projects/2017/glue/notebook.md]: a general point, I think the "desc" field should be a little longer here, just looking at the way they render on the website, this is your sales pitch, it's all people see without clicking on the idea.

Comment (astrofrog) [inline _projects/2017/glue/notebook.md]: Ah ok, wasn't sure - can make it longer :)

**Pull request merged**